kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/document/validation/configuration/RequisitionValidation.xml

```diff
@@ -50,5 +50,14 @@
       </list>
     </property>
   </bean>
+  

```

Comment:
The org.kuali version of RequisitionValidation.xml already has the PurchasingAccountsPayable-itemTypeTaxPolicyValidation bean defined to the route document event (see line 80).

If you had to provide this override in order to make this Requistion validation work, I suggest flipping lines 30 and 31 in edu.arizona.kfs.module.purap.spring-purap.xml. PurapValidatorDefinitions.xml should be first in the list of validators.